exporter/sumologicexporter/config.go

```diff
@@ -0,0 +1,66 @@
+// Copyright 2019, OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package sumologicexporter
+
+import (
+	"go.opentelemetry.io/collector/config/configmodels"
+)
+
+// Config defines configuration for Sumo Logic exporter.
+type Config struct {
+	configmodels.ExporterSettings `mapstructure:",squash"` // squash ensures fields are correctly decoded in embedded struct.
+
+	// Global options
+	// Unique URL generated for your HTTP Metrics Source.
```

Comment:
The exporter config should support all three pillars. So I think we should just have the generic URL field here